d2/src/main/java/com/linkedin/d2/balancer/clients/TrackerClientImpl.java

```diff
@@ -62,6 +65,8 @@
 {
   public static final String DEFAULT_ERROR_STATUS_REGEX = "(5..)";
   public static final Pattern DEFAULT_ERROR_STATUS_PATTERN = Pattern.compile(DEFAULT_ERROR_STATUS_REGEX);
+  public static final List<HttpStatusCodeRange> DEFAULT_ERROR_STATUS_RANGES = Arrays.asList(new HttpStatusCodeRange()
```

Comment:
same thing defined in Relative...Strategy

Reply:
Which line in RelativeLoadBalancerStrategy?